xbmc/SortFileItem.h

```diff
@@ -19,51 +19,61 @@
  *
  */
 
+///
+/// \defgroup python_xbmcplugin_sort_method List Item sort methods.
+/// \ingroup python_xbmcplugin
+/// @brief List of sort methods
+///
+/// Here are the Python integer constant names listed and the related integer
+/// value which are used on xbmcplugin.addSortMethod(...).
+///
+///@{
 typedef enum {
-  SORT_METHOD_NONE=0,
-  SORT_METHOD_LABEL,
-  SORT_METHOD_LABEL_IGNORE_THE,
-  SORT_METHOD_DATE,
-  SORT_METHOD_SIZE,
-  SORT_METHOD_FILE,
-  SORT_METHOD_DRIVE_TYPE,
-  SORT_METHOD_TRACKNUM,
-  SORT_METHOD_DURATION,
-  SORT_METHOD_TITLE,
-  SORT_METHOD_TITLE_IGNORE_THE,
-  SORT_METHOD_ARTIST,
-  SORT_METHOD_ARTIST_AND_YEAR,
-  SORT_METHOD_ARTIST_IGNORE_THE,
-  SORT_METHOD_ALBUM,
-  SORT_METHOD_ALBUM_IGNORE_THE,
-  SORT_METHOD_GENRE,
-  SORT_METHOD_COUNTRY,
-  SORT_METHOD_YEAR,
-  SORT_METHOD_VIDEO_RATING,
-  SORT_METHOD_VIDEO_USER_RATING,
-  SORT_METHOD_DATEADDED,
-  SORT_METHOD_PROGRAM_COUNT,
-  SORT_METHOD_PLAYLIST_ORDER,
-  SORT_METHOD_EPISODE,
-  SORT_METHOD_VIDEO_TITLE,
-  SORT_METHOD_VIDEO_SORT_TITLE,
-  SORT_METHOD_VIDEO_SORT_TITLE_IGNORE_THE,
-  SORT_METHOD_PRODUCTIONCODE,
-  SORT_METHOD_SONG_RATING,
-  SORT_METHOD_SONG_USER_RATING,
-  SORT_METHOD_MPAA_RATING,
-  SORT_METHOD_VIDEO_RUNTIME,
-  SORT_METHOD_STUDIO,
-  SORT_METHOD_STUDIO_IGNORE_THE,
-  SORT_METHOD_FULLPATH,
-  SORT_METHOD_LABEL_IGNORE_FOLDERS,
-  SORT_METHOD_LASTPLAYED,
-  SORT_METHOD_PLAYCOUNT,
-  SORT_METHOD_LISTENERS,
-  SORT_METHOD_UNSORTED,
-  SORT_METHOD_CHANNEL,
-  SORT_METHOD_CHANNEL_NUMBER,
-  SORT_METHOD_BITRATE,
-  SORT_METHOD_DATE_TAKEN,
-  SORT_METHOD_MAX
+  SORT_METHOD_NONE=0,                             //!< Integer: __0__ - Do not sort
+  SORT_METHOD_LABEL,                              //!< Integer: __1__ - Sort by label
+  SORT_METHOD_LABEL_IGNORE_THE,                   //!< Integer: __2__ - Sort without the label
```

Comment:
Well if you can see ```SORT_METHOD_LABEL_IGNORE_THE``` it's not that bad as it explains what it does. 
``` Sort without the label``` is confusing better might be ``` Sort by label without 'the'```